rotkehlchen/tasks/utils.py

```diff
@@ -0,0 +1,24 @@
+from typing import TYPE_CHECKING
+from rotkehlchen.constants.timing import DATA_UPDATES_REDETECT
+from rotkehlchen.db.updates import LAST_DATA_UPDATES_KEY
+from rotkehlchen.serialization.deserialize import deserialize_timestamp
+from rotkehlchen.utils.misc import ts_now
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def should_check_data_updates(database: 'DBHandler') -> bool:
```

Comment:
Both you and Alexey have the same file here, with the same thing, but just checkign a different constant. This should be abstracted by one of you. Guess whoever handles the conflicts.